.github/workflows/run-tests-on-push.yml

```diff
@@ -19,6 +25,11 @@ jobs:
         if: runner.os == 'Linux'
       - run: npm test
         if: runner.os != 'Linux'
+      - name: Publish
+        if: success() && startsWith(github.ref, 'refs/tags/')
```

Comment:
Does this limit the deploy/publish actions to certain branches or tags? I'm realizing that we could use GitHub [releases](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases), but that might not be what we are going for.
(this might be a dumb question as I've not configured GitHub actions)

Reply:
Awesome, sounds good to me!